backend/templates/emails/renewal_reminder.html

```diff
@@ -0,0 +1,57 @@
+{% extends 'emails/base.html' %}
+
+{% block content %}
+<h2>Reminder to Renew <b>{{ name }}</b> on Penn Clubs & SAC Fair Signup by 8/24!</h2>
+<p style="font-size: 1.2em">
+    Dear <b>{{ name }}</b> leaders,
+</p>
+
+<p style="font-size: 1.2em; color: #8B2222">
+    You are receiving this notification because you were the contact for your organization during the 2019 SAC
+    Activities Fair. You have not yet renewed your club for the {{ year }} school year.
+    <b>
+        If you do not submit a renewal request by August 24th, your club may not be approved by the Office of
+        Student Affairs or participate in the SAC virtual Activities Fair for new student orientation.
```

Comment:
@lantingchiang Minor nitpick, could you rename this to "SAC Virtual Activities Fair"? Just capitalize the "V".

Other than that, I'm happy with this email template. :)